### PR TITLE
Gives Dionae a speedbuff

### PR DESCRIPTION
**Pull request by Yonnimer**
What it says in the title. This changes dionae speed from 7 to 5. In full light, this will make them about the speed of a xion, if not a little slower. Given they're usually not in full light, however, they should be closer to a G1's speed as evident by the testing done.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: html/changelogs/yonnimer-speed.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rtweak: "Gives dionae a speedbuff, changing their slowdown from 7 to 5.
```
Comment (Geevies) [inline html/changelogs/yonnimer-speed.yml:41]:
```suggestion
  - tweak: "Gives dionae a speedbuff, changing their slowdown from 7 to 5."
```

Comment (Yonnimer) [reply html/changelogs/yonnimer-speed.yml]: fixed

File: html/changelogs/yonnimer-speed.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Gives dionae a speedbuff, changing their slowdown from 7 to 5.
```
Comment (Geevies) [inline html/changelogs/yonnimer-speed.yml:41]:
```suggestion
  - tweak: "Gives dionae a speedbuff, changing their slowdown from 7 to 5."
```

**Review by Geevies — CHANGES_REQUESTED** [review]

**Review by Chada-1 — APPROVED** [review]

**Comment by Arrow768** [comment]
Will be discussed between the maintainers.

**Comment by Arrow768** [comment]
Closed as not wanted after discussion with the maintainers.
The reasoning has been explained to the diona lore writers in the staff discord.

**Pull request closed without merging**